specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/stable/2018-06-01/examples/TopQueryStatisticsListByServer.json

```diff
@@ -0,0 +1,64 @@
+{
+  "parameters": {
+    "api-version": "2018-06-01-privatepreview",
+    "subscriptionId": "ffffffff-ffff-ffff-ffff-ffffffffffff",
+    "resourceGroupName": "testResourceGroupName",
+    "serverName": "testServerName",
+    "parameters": {
+      "properties": {
+        "numberOfTopQueries": 5,
+        "aggregationFunction": "avg",
+        "observedMetric": "duration",
+        "observationStartTime": "2019-05-01T20:00:00.000Z",
+        "observationEndTime": "2019-05-07T20:00:00.000Z",
+        "aggregationWindow": "PT15M"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/resourceGroups/testResourceGroupName/providers/Microsoft.DBforPostgreSQL/servers/testServerName/queryStatistic/66-636923268000000000-636923277000000000-avg-duration",
```

Comment:
these are "topQueryStatistics", not "queryStatistic"

Reply:
WIll fix this

Reply:
If a user requests a topQueryStatistics resource type they need to get that back.  This will result in s360 violations on your RP.  See https://armwiki.azurewebsites.net/api_contracts/RPCstrongvalidation.html?q=validation

Reply:
Could you suggest some workaround here, this is an APi in production for the last 2 years and it has always returned queryStatistics as the response object. The idea was to return the top from the list of querystatistics.
@ssen-msft FYI

Reply:
Please mark it down on your end to fix in the next api-version

Reply:
Noted, will create a workitem at our end